### PR TITLE
Change confirmation prompt on trending management

### PR DESCRIPTION
**Pull request by tribela**
Same approach as #19411, Confirmation prompt should be different on each actions

### CODE REVIEW

**Review by antimundo — APPROVED** [review]

File: config/locales/en.yml [hunk 1]
```diff
@@ -760,10 +760,16 @@ en:
     trends:
       allow: Allow
       approved: Approved
+      confirm_allow: Are you sure to allow selected tags?
```
Comment (TobyWilkes) [inline config/locales/en.yml:763]:
```suggestion
      confirm_allow: Are you sure you want to allow selected tags?
```

Comment (TobyWilkes) [reply config/locales/en.yml:763]: Unless I'm mistaken the syntax needs to be "Are you sure you want", if its to replace the admin.reports.are_you_sure message.

**Comment by mjankowski** [comment]
Is there any way we could have some automatic insertion of the surrounding JS context into these values, instead of needing to add i18n values for every single interaction?

**Review by mjankowski — COMMENTED** [review]
This needs rebase, but is probably fine ... or at least along the lines as the other similar/merged PR.

That said ... how do we feel about the trade-off here on getting this (admittedly slightly nicer!) specialized message vs the generic confirm RE: the i18n maintenance burden? Definitely seems sligthly better, but maybe not enough to justify?

**Review by ThisIsMissEm — APPROVED** [review]
This looks good to me as a change.

**Comment by renchap** [comment]
Looks good, but could you remove the Korean translation file?

All translation files, except the english one, are managed through Crowdin, and will be overridden.

**Comment by tribela** [comment]
> Looks good, but could you remove the Korean translation file?
> 
> All translation files, except the english one, are managed through Crowdin, and will be overridden.



> Looks good, but could you remove the Korean translation file?
> 
> All translation files, except the english one, are managed through Crowdin, and will be overridden.

Done! 😄

**Review by renchap — APPROVED** [review]

**Pull request merged**